src/main/java/com/spikes2212/control/FeedForwardController.java

```diff
@@ -76,6 +76,13 @@ public void setGains(double kS, double kV, double kA, double kG) {
         this.kG = kG;
     }
 
+    public void setGains(FeedForwardSettings feedForwardSettings) {
+        this.kS = feedForwardSettings.getkS();
```

Comment:
instead of this, use `setGains(feedForwardSettings.getkS(), feedForwardSettings.getkV()...)`